app/views/claims/student_loan.html.erb

```diff
@@ -10,14 +10,11 @@
           <fieldset class="govuk-fieldset" aria-describedby="student_loans-hint" role="group">
             <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
               <h1 class="govuk-fieldset__heading">
+                <span class="govuk-caption-xl">About your student loans</span>
```

Comment:
Do you think we should have the caption outside of the `h1`? The design system seems to suggest if it's a section name it should be outside and if it's part of the heading it should be inside. My guess is there's a semantic difference for assistive tech.

https://design-system.service.gov.uk/styles/typography/#headings-with-captions

Reply:
Ah, I misread that. Lemme sort that out 👍